src/internal/operators/retry.ts

```diff
@@ -63,22 +76,10 @@ export interface RetryConfig {
  *
  * @see {@link retryWhen}
  *
- * @param count - Number of retry attempts before failing.
- * @param resetOnSuccess - When set to `true` every successful emission will reset the error count
```

Comment:
This parameter doesn't exist anymore in this form, so I removed the docs for it. It is now part of the RetryConfig and it's properly documented.